ui/components/component-library/checkbox/checkbox.scss

```diff
@@ -13,6 +11,7 @@
 
     &:hover:not(:disabled) {
       background-color: var(--color-background-default-hover);
+      cursor: pointer;
```

Comment:
Will this work when the label is present as well? We may need it in both places. Just waiting for storybook build to test